awswrangler/_utils.py

```diff
@@ -99,8 +99,11 @@ def _get_endpoint_url(service_name: str) -> Optional[str]:
         endpoint_url = _config.config.dynamodb_endpoint_url
     elif service_name == "secretsmanager" and _config.config.secretsmanager_endpoint_url is not None:
         endpoint_url = _config.config.secretsmanager_endpoint_url
-    elif service_name == "timestream" and _config.config.timestream_endpoint_url is not None:
+    elif service_name == "timestream-write" and _config.config.timestream_endpoint_url is not None:
```

Comment:
The same change is required here: https://github.com/aws/aws-sdk-pandas/blob/timestream-write/awswrangler/_utils.py#L83

Thanks

Reply:
The link you shared seems to be this same function here where the change is.

Reply:
Ah yes my bad, [here](https://github.com/aws/aws-sdk-pandas/blob/timestream-write/awswrangler/_config.py#L51) rather. We might need to set two different endpoint urls, one for write and the other for query

Reply:
Gotcha, yea agreed we should allow for setting both. Will update.